instructions/instructions.css

```diff
@@ -0,0 +1,126 @@
+html {
+  scroll-snap-type: x mandatory; /* to swipe between pages*/
+}
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  background: #fffaf4;
+  font-family: IBM Plex Sans;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.title {
+  font-size: 23px;
+  padding: 51px;
+  margin-bottom: 2px;
+  padding-bottom: 51px;
+  font-family: IBM Plex Sans;
+  font-style: normal;
+  line-height: 24px;
+  text-align: center;
+  color: #000000;
+  font-weight: bold;
+  margin-top: -7px;
+}
+.parentDiv {
+  background-image: url(../images/background_img.png);
+  background-repeat: no-repeat;
+  background-size: 100%;
+
+  scroll-snap-align: center; /* to swipe between pages*/
+  scroll-snap-stop: always; /* to swipe between pages*/
+}
+.img {
+  margin-left: 0px;
+}
+.num {
+  width: 105px;
+  height: 26px;
+  left: 128px;
+  top: 475px;
+  font-family: IBM Plex Sans;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 50px;
+  line-height: 61px;
+  text-align: center;
+  font-weight: bold;
+  color: #df3030;
+  margin-bottom: 66px;
+  margin-top: 50px;
+}
+
+.step {
+  margin-left: 0px;
+  font-family: IBM Plex Sans;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 22px;
+  line-height: 26px;
+  text-align: center;
+  color: #10242c;
+  font-weight: bold;
+}
+
+.para {
+  font-family: IBM Plex Sans;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 21px;
+  text-align: center;
+  color: #707a7c;
+  padding-left: 25px;
+  padding-right: 25px;
+  margin: 20px;
+}
+
+.bar {
+  margin-left: 2px;
+  margin-top: 15px;
+}
+
+.skip-btn {
+  margin-top: 88px;
+  margin-right: 261px;
+
+  width: 33px;
+  height: 12px;
+  left: 44px;
+  top: 700px;
+  font-family: IBM Plex Sans;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 70.5%;
+  text-align: center;
+  border: none;
+  color: #000000;
+  background-color: #fffaf4;
+  cursor: pointer;
+}
+
+/**/
+.center {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: -19px;
+}
+.flex-colum {
+  flex-direction: column;
+}
+/* SWIP SECTION*/
+.container {
+  display: flex;
```

Comment:
There is a lot of scroll due to overflow in your flexbox container
Try add this to `.container` class 
`  overflow:auto;
scroll-snap-type: x mandatory; 
`